src/metalkit.cs

```diff
@@ -288,6 +297,20 @@ interface MTKTextureLoader {
 		[Async]
 		void FromName (string name, nfloat scaleFactor, [NullAllowed] NSBundle bundle, [NullAllowed] MTKTextureLoaderOptions options, MTKTextureLoaderCallback completionHandler);
 
+		[NoiOS]
+		[NoTV]
+		[Mac (10,12, onlyOn64 : true)]
+		[Export ("newTextureWithName:scaleFactor:displayGamut:bundle:options:completionHandler:")]
+		[Async]
+		void FromName (string name, nfloat scaleFactor, NSDisplayGamut displayGamut, [NullAllowed] NSBundle bundle, [NullAllowed] NSDictionary options, MTKTextureLoaderCallback completionHandler);
+
+		[NoiOS]
+		[NoTV]
+		[Mac (10,12, onlyOn64 : true)]
+		[Wrap ("FromName (name, scaleFactor, displayGamut, bundle, options == null ? null : options.Dictionary, completionHandler)")]
```

Comment:
Use `options?.Dictionary` next time easier to read than `options == null ? null : options.Dictionary` :)

Reply:
Copy and pasting the code above was just too easy though :P